### ISSUE TITLE
Is it intended that the unit test are published to npm?

### ISSUE THREAD

**Issue by erikerikson** [aws/aws-sdk-js#2247]
TL;DR:
Unit tests were published as part of `aws-sdk@2.311.0`.  Was that intended?

It is non-problematic but we have a travisci job that had legacy config (we are updating this) targeting the nodejs v4.10 runtime.  A recent PR triggered the download and use of [`aws-sdk@2.311.0`](https://travis-ci.org/Nordstrom/hello-retail/builds/425969842#L782).  That in turn caused the test code at `aws-sdk/scripts/lib/test-helper.spec.js` to be loaded which, using destructuring, [failed interpretation](https://travis-ci.org/Nordstrom/hello-retail/builds/425969842#L1085) and failed the build.

This is **NOT** a request to support a non-LTS version (that seems like it would be a bad choice).  I am happy that the build failed and spurned me to update target runtimes (for my lambdas as well).  Given that these sorts of things are generally not published I am instead opening this to ask: was publishing your unit tests (.spec.js files) intentional?

**Comment by chrisradek** [comment]
@erikerikson 
Thanks for reporting this. We did not intend to publish test files to NPM, though I can see why we missed them in this case. We'll update our `.npmignore` file to make sure they aren't published in the future.

**Comment by srchase** [comment]
This has been fixed.  Closing this issue.
